content/git/index.md

```diff
@@ -78,17 +106,21 @@ Better Example:
 
 * Don't split a single *logical change* into several commits. For example,
   the implementation of a feature and the corresponding tests should be in the
-  same commit. Changes to a precompiled style resource and the resulting compiled `.css` should be in the same commit.
+  same commit.
 
 * Commit *early* and *often*. Small, self-contained commits are easier to
   understand and revert when something goes wrong.
 
 * Commits should be ordered *logically*. For example, if *commit X* depends
   on changes done in *commit Y*, then *commit Y* should come before *commit X*.
 
+* 🤓 [rebase](https://git-scm.com/docs/git-rebase) and [--amend](https://git-scm.com/docs/git-commit#Documentation/git-commit.txt---amend)
+  are useful commands that can help you maintain a clean commit history on your
+  _personal_ working branches. Learn them to save time and frustration!
+
 ### Messages
 
-* Use the editor, not the terminal, when writing a commit message:
+* Use the editor or a git GUI (e.g. [SourceTree](https://www.sourcetreeapp.com/) and [GitKraken](https://www.gitkraken.com/)), not the terminal, when writing a commit message:
```

Comment:
This is a personal thing but I don't know that I agree with this one. I only ever write commit messages from the terminal and even VS Code has a small input field for the commit messages, which lends towards brevity.

Reply:
I think it comes down to personal preference. I don't prefer editing in Vim, and SourceTree gives you a big textarea for your commit message:
![image](https://user-images.githubusercontent.com/1933201/131850929-b6da83f8-5b1d-4ba0-97cb-b20479e8a604.png)

Reply:
I think this bullet could be clarified, though. It's saying "not the terminal" – but I believe the "editor" in reference is Vim.

Reply:
I've removed the guidance of what tool to use when writing a commit message, and moved the focus to the content of the message.